File: mvvm/src/main/java/com/lee/mvvm/base/constract/IBaseView.java
package com.lee.mvvm.base.constract;

import com.lee.mvvm.utils.LifeCycleHelper;

public interface IBaseView {
    /**
     * 获取默认的LifeCycleHelper
     *
     * @return {@link LifeCycleHelper}
     */
    LifeCycleHelper getDefaultLifeCycle();
}

File: mvvm/src/main/java/com/lee/mvvm/base/view/intf/IBaseView.kt
package com.lee.mvvm.base.view.intf

import com.lee.mvvm.utils.LifeCycleHelper

interface IBaseView {
    /**
     * 获取默认的LifeCycleHelper
     *
     * @return [LifeCycleHelper]
     */
    fun getDefaultLifeCycle() : LifeCycleHelper
}
